test/e2e/e2e_suite_test.go

```diff
@@ -22,25 +22,25 @@ const (
 )
 
 var (
-	// default
-	Platform    = "kind"
-	K8sVersion  = "v1.17.17"
```

Comment:
This was stale non used code right? we control the kind version outside for CI and shell AFAIK.

Reply:
Exactly